lalrpop/src/lr1/error/mod.rs

```diff
@@ -371,6 +386,36 @@ impl<'cx, 'grammar> ErrorReportingCx<'cx, 'grammar> {
             .end()
     }
 
+    fn report_error_ambiguous_reduction(
+        &self,
+        reduce: Example,
+        span1: Span,
+        span2: Span,
+    ) -> Message {
+        let file_text = Tls::file_text();
+        let styles = ExampleStyles::new();
+        let span1_str = file_text.span_text(span1);
+        let span2_str = file_text.span_text(span2);
+        MessageBuilder::new(span1)
+            .heading()
+            .text("Multiple productions for the same reduction")
+            .end()
+            .body()
+            .begin_lines()
+            .wrap_text(format!(
+                "The following symbols can be reduced into a {} in two ways",
+                reduce.reductions.first().unwrap().nonterminal
+            ))
+            .push(reduce.to_symbol_list(reduce.symbols.len(), styles))
+            .wrap_text("They could be reduced using the following production:")
+            .wrap_text(span1_str)
```

Comment:
TODO: I'd like to have this point at line numbers too.  Right now it just includes the text at the location.  Line numbers will be particularly helpful if the user has written the exact same thing twice.